.github/renovate.json5

```diff
@@ -202,4 +202,4 @@
     },
   ],
   platformAutomerge: true,
-}
\ No newline at end of file
+}
```

Comment:
please run `python3 hack/build/ci/update-renovate-json5.py` to reformat file properly to avoid dummy new line automatic PR changes.